### PR TITLE
Fix: Adding comment in pr template 

### LINKED ISSUE #445: 🐛 Bug: "Enhancement" Issue Template field needs adjusting
### Describe the bug

The "additional context" field should be a multiline text field like the others.

### Steps To Reproduce

Start the process to create a new "enhancement" issue.

### Expected Behavior

The "additional context" field should be a multiline text field like the others.

### Screenshots

image

### Device Information [optional]

```markdown
- OS:
- Browser:
- version:
```

### Are you working on this issue?

No

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
**What kind of change does this PR introduce?**

Added a comment in pr template so that user creating a new only chooses the relevant option.

**Issue Number:**
-  Closes #445  
-  Related to #436  
-  Others? 

**Screenshots/videos:**
N/A

**If relevant, did you update the documentation?**
N/A

**Summary**
Added comment in the pr template.

**Does this PR introduce a breaking change?**
N/A

### CODE REVIEW

**Comment by DhairyaMajmudar** [comment]
@benjagm pls. have a look.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Thanks for submitting a pull request! Please provide enough information so that
 <!-- E.g. a bugfix, feature, refactoring, etc… -->
 
 **Issue Number:**
+<!-- Pick one of the below option -->
```
Comment (benjagm) [inline .github/pull_request_template.md:13]:
```suggestion
<!-- Pick one of the below options -->
```

**Review by benjagm — COMMENTED** [review]
Just a small suggestion.

File: .github/pull_request_template.md (hunk 1 above)
Comment (DhairyaMajmudar) [reply .github/pull_request_template.md]: Commited the suggestions : )

**Review by benjagm — APPROVED** [review]
LGTM

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Thanks for submitting a pull request! Please provide enough information so that
 <!-- E.g. a bugfix, feature, refactoring, etc… -->
 
 **Issue Number:**
+<!-- Pick one of the below options -->
```
Comment (gregsdennis) [inline .github/pull_request_template.md:13]:
```suggestion
<!-- Pick one of the below options.  Please remove those which don't apply. -->
```

I really want to call out that they should only include the items which make sense.  Less information for the PR reader to parse is better.

**Review by gregsdennis — APPROVED** [review]
What there is fine.  Maybe a small push as well?

An alternative would be to put all of the options as comments so that they have to uncomment the one(s) they want.

**Comment by DhairyaMajmudar** [comment]
> What there is fine. Maybe a small push as well?
> 
> An alternative would be to put all of the options as comments so that they have to uncomment the one(s) they want.

Thank you for review, I have pushed the suggested changes.

**Comment by benjagm** [comment]
Just to confirm. Is this ready to merge? Any change pending from your side @DhairyaMajmudar ?

**Comment by DhairyaMajmudar** [comment]
Yes pr is ready to merge : )

**Review by chual5614-code — COMMENTED** [review]
Đl

**Pull request merged**